### ISSUE TITLE
"Wallet loaded, but with errors..."

### ISSUE THREAD

**Issue by RustinBbu** [unchained-capital/caravan#235]
Creating and loading 2/3 and 3/5 wallets gives me "Wallet loaded, but with errors..." warning, and funds sent to the receive address are not showing as available, despite being confirmed on the blockchain.

* [ ] Regression (a behavior that used to work and stopped working in a new release)
* [ ] Bug report 
* [ ] Feature request
* [ ] Documentation issue or request
* [x] Support request

## Expected Behavior

I expected that scanning the QR code for receive address and seeing the transaction with 6+ confirmation on the blockchain should show available funds in the UTXO.

## Current Behavior

I created a 3/5 wallet with 5 Trezor One's. Upon creating the wallet, I got a message that read in red text, ""Wallet loaded, but with errors...". I then used 3 of those same Trezors to create a new 2/3 wallet. Same message. To test these wallets out, I sent some bitcoin to each of the wallets by scanning the QR code in the "Receive" tab. Both transactions to the 2/3 and 3/5 wallet confirmed from the sender side, but I don't see any funds in either wallet.

I checked the blockchain and see that the funds were sent to the correct address, which is still listed as the first address in the wallet. However, the address shows zero funds available. The transaction was sent from a Blue Wallet wallet which is connected to my own node. This was several days ago.

The configuration I used was P2SH/Mainnet/Public/Default. I also downloaded the wallet configuration for each wallet, reloaded the wallets, and the error persisted, plus no funds available.

## Steps to Reproduce (for bugs)

Screenshot: https://www.screencast.com/t/u7AIErRyMlOq

* Where are you running caravan:   unchained-capital.github.io
* Operating system:  Mac
* Browser and version:  Chrome

**Comment by humanumbrella** [comment]
Hmm ... that's odd. So just to confirm - the address shown has a balance on a block explorer but it's not showing up in caravan using the "Public" node option, correct?

Are you using standard/default BIP32 paths?

The error message seems to suggest that the public node is not behaving properly. E.g. there are errors when trying to fetch UTXOs for a particular address from the node.

**Comment by RustinBbu** [comment]
Your brainstorming helped fix it, so thank you!

Even though Caravan was not connected to my private node during the wallet creation process, I was running the Caravan Multisig Tool on MyNode. Once I disabled that and reloaded the wallet configuration, the funds now appear normally, and the wallets load without the error message.

I guess the error was that I created the wallet with a public node, but it was attempting to connect to my private node. 

I hope documenting this helps someone in the future solve a similar problem.
